bentoml/_internal/io_descriptors/pandas.py

```diff
@@ -108,7 +108,7 @@ def infer(input_df: pd.DataFrame) -> pd.DataFrame:...
     def __init__(
         self,
         orient: Literal[
-            "dict", "list", "series", "split", "records", "index"
+            'split', 'records', 'index', 'columns', 'values', 'table'
```

Comment:
@aarnphm actual valid orients for DataFrames

Reply:
you can't input other type of orient since it is a Literalt type right?

Reply:
yeah. my point is, 'dict' & 'list' are not valid orients